packages/atlas/src/components/_nft/NftWidget/NftWidget.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from '@emotion/styled'
+import { Meta, Story } from '@storybook/react'
+import { add } from 'date-fns'
+import React from 'react'
+import { BrowserRouter } from 'react-router-dom'
+
+import { NftWidget, NftWidgetProps } from '.'
+
+export default {
+  title: 'NFT/NFT Widget',
+  component: NftWidget,
+  argTypes: {
+    size: {
+      control: { type: 'select', options: ['small', 'medium'] },
+    },
+    status: {
+      control: { type: 'select', options: ['iddle', 'buy-now', 'auction'] },
+    },
+    nftState: { table: { disable: true } },
+  },
+  args: {
+    ownerHandle: 'ye 🖤',
+    ownerAvatarUri: 'https://54.172.37.177.nip.io/distributor-1/api/v1/assets/3',
```

Comment:
Maybe we use some other URL? This one may become dead once we switch to newer playground